client/src/config/example-cooking-apiKey.txt

```diff
@@ -1 +0,0 @@
-export const apiKey = 'your api key';
```

Comment:
I'd suggest we keep this file for now as I see it as a useful resource for project newcomers. Should we wish to remove it, we need to create a separate ticket just for that.